src/settings-ui/Settings.UI/SettingsXAML/Views/AlwaysOnTopPage.xaml

```diff
@@ -7,6 +7,8 @@
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:ui="using:CommunityToolkit.WinUI"
+    xmlns:viewmodels="using:Microsoft.PowerToys.Settings.UI.ViewModels"
+    d:DataContext="{d:DesignInstance Type=viewmodels:AlwaysOnTopViewModel}"
```

Comment:
@jay-o-way Specific reason to add these? Typically you'd add these to provide a `DataContext` during design time (hence the `d` / `DesignInstance`). That means you could show data in the Visual Studio or Expression Blend designer.

However, we use WinUI 3 that has no designer support at all in Visual Studio..

Reply:
Visual Studio keeps suggesting it 😅 says it might make Intellisense faster, too.

Reply:
Does it link to any docs? It's weird, because it's not there by default and doesn't make sense without having an ctual designer. I'd just remove it so we keep namespaces clean.. might be a couple of other ones that are not needed (VS will grey those out)

Reply:
> might be a couple of other ones that are not needed

That's totally true. Outside scope for this pr and even for the other one. If somebody searches for unneeded namespaces, check the connected _.xaml.cs_ files too.

Reply:
I think those will just be ignored.. and are often left when code gets removed to another file or>

On the `designinstance` namespaces, let's get rid of those as these don't add anything and will only break compilation when e.g. a viewmodel name changes.

Reply:
Will do.